dev-docs/bidders/lm_kiviads.md

```diff
@@ -2,7 +2,7 @@
 layout: bidder
 title: LM KiviAds
 description: LM Kiviads Bidder Adapter
-biddercode: xe
+biddercode: lm_kiviads
```

Comment:
@KiviAds - I don't see a bidder with this name. Your Prebid.js adapter's bidder code is `kiviads`... what is the 'LM' part?